topics/ajax/index.md

```diff
@@ -0,0 +1,9 @@
+---
+display_name: Ajax
+short_description: Ajax is a web technology used by web apps and web pages to exchange data in the background.
```

Comment:
This is inaccurate. "Ajax is not a single technology, but rather a group of technologies." [Source](https://en.wikipedia.org/wiki/Ajax_%28programming%29).